brave-grpc/README.md

```diff
@@ -0,0 +1,20 @@
+# brave-jersey #
+
+Contributed by [Eric Hauser](https://github.com/ewhauser).
+
+Contains a client and server interceptor for [grpc-java](github.com/grpc/grpc-java) that integrates Brave with gRPC.
+
+To enable tracing for a gRPC application, add the interceptors when when constructing the client and server bindings:
+
+```java
+    Server server = ServerBuilder.forPort(serverPort)
+        .addService(ServerInterceptors.intercept(
+            GreeterGrpc.bindService(new GreeterImpl()), new BraveGrpcServerInterceptor(brave)))
+        .build()
+        .start();
+        
+    ManagedChannel channel = ManagedChannelBuilder.forAddress("localhost", serverPort)
+        .intercept(new BraveGrpcClientInterceptor("helloCallingService", brave))
+        .usePlaintext(true)
+        .build();
+```
```

Comment:
this might need a development section, showing how to make intellij compile the protos